Neos.Media.Browser/composer.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "neos/media-browser",
+    "type": "typo3-flow-package",
+    "description": "Add description here",
```

Comment:
This module allows managing of media assets including pictures, videos, audio and documents.